backstop_data/engine_scripts/puppet/search/find-activities-delete-modal.js

```diff
@@ -6,5 +6,8 @@ module.exports = async (engine, scenario, vp) => {
   const page = await Page.build(engine, scenario, vp);
 
   await require('./find-activities')(engine, scenario, vp);
+  if( !!(await engine.$('.crm-search-results span.crm-hover-button'))) {
```

Comment:
do you have the linter running? The style here is wrong
```js
await require('./find-activities')(engine, scenario, vp);

if (!!(await engine.$('.crm-search-results span.crm-hover-button'))) {
  await engine.click('.crm-search-results span.crm-hover-button');
}

await page.clickAndWaitForModal('a[title="Delete Activity"]');
```
please check that the code is clean before sending it to review

Reply:
Oops! Missed that. Fixing.